src/Router.jsx

```diff
@@ -0,0 +1,13 @@
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Market from './pages/Market/Market';
+function Router() {
+  return (
+    <BrowserRouter>
+        <Routes>
+            <Route path="/item" element={<Market />}/>
```

Comment:
P2:
요구사항에는 `중고마켓 페이지 주소는 “/items” 입니다.`로 되어 있으니 아래처럼 하시면 좋겠습니다.
```suggestion
            <Route path="/items" element={<Market />}/>
```